pkgs/development/python-modules/numpy/default.nix

```diff
@@ -50,7 +51,13 @@ in buildPythonPackage rec {
     sha256 = "1za22ggjxzm7drs2vd15s81ad9rlshk4p7pv7mxcbz4acdiszx6z";
   };
 
-  patches = lib.optionals python.hasDistutilsCxxPatch [
+  patches = [
+    # fix https://github.com/numpy/numpy/issues/19624
+    (fetchpatch {
+      url = "https://github.com/numpy/numpy/commit/ae279066d6bd253e8675428fac8946938b8d48d9.diff";
+      sha256 = "sha256-q8SoF/pMzLI4zYCx2YAZzR31cARlZXerNnBSevBO5pE=";
```

Comment:
`sha256 =` could be replaced with `hash =` here.

Reply:
That wouldn't work yet
https://github.com/NixOS/nixpkgs/blob/722c8c9529baadd73fb0e5cf1ec896ca2a90b8f3/pkgs/build-support/fetchpatch/default.nix#L16

Reply:
Oh we should add `hash` support to fetchpatch then